### DISCUSSION TITLE
Activity Widget Rendering Glitched

### DISCUSSION THREAD

**Discussion by Xevion** [community/community#141021] [Mobile]
![Screenshot_20241009-170018.png](https://github.com/user-attachments/assets/e0b950df-ce7e-4c4d-ad83-c68141229963)

Doesn't have a specific trigger as far as I understand. No other issues with widgets in my experience. Sometimes doesn't happen, but lately it's always like that, light and dark mode.

Google Pixel 7 Pro
Nova Launcher
Android 14

**Comment by uzzal5803** [comment]
Give me chances

**Comment by stevepopovich** [comment]
Hi @Xevion so you are saying this is stuck like this now?

**Reply by Xevion** [reply]
I am unsure what you mean by that question. It becomes stuck like this sometimes, sometimes it's not like that (renders normally).

**Reply by stevepopovich** [reply]
What are the conditions that causes it? Is it after resizing? Or you just come to the home screen and it looks like this? Then you come back later and it looks normal?

**Reply by Xevion** [reply]
So far I haven't noticed any particular trigger. I don't resize, move, or change settings for it, or any other widgets. I just go to my screen, and the widget is messed up like that. Usually once it happens, it stays like that indefinitely (until something resets it - perhaps a restart/refresh/reload of some kind).

**Reply by stevepopovich** [reply]
Gotcha. Thank you very much for that feedback. I will be investigating this internally and come back with a fix—hopefully soon.

**Reply by stevepopovich** [reply]
@Xevion are you still seeing this problem?

**Reply by Xevion** [reply]
Yes. I notice when I resize it or change the padding, it goes to a reasonably correct-looking state, and then a moment later returns to a weird design with way too much padding.

**Reply by Xevion** [reply]
It does look different, though. I could almost be convinced that it's just bad design...

![Screenshot_20250620-143543](https://github.com/user-attachments/assets/743e5528-96e5-41cd-bda8-1fdb42e0f0ae)

**Reply by stevepopovich** [reply]
No, that is a def a bug :) We know what is happening, but it is a tricky thing to fix. We will try to look sooner rather than later. 

This happens because the widget is "too big" so if you make it smaller, it might help. You can also "reinstall" the widget, we have made some updates that might help. But yeah, those are less than ideal workarounds and this is our bad. We will let you know when we have a proper fix.

**Reply by Xevion** [reply]
Good to hear you're still actively working towards it. I tried messing around with the size again, I made a recording to show you or whoever else might want to see it. Nothing I do seems to get it into a 'legitimate' state.

https://github.com/user-attachments/assets/dcb75d6c-d731-49f3-b46d-2f70a00add0e

**Reply by stevepopovich** [reply]
That is...interesting and unexpected. We will try to dig a bit more. Thank you very much for the recording!
